src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const oobjectified = sourceString
+    .split(';')
+    .map(prop => prop
+      .trim()
+      .split(':')
+      .map(item => item.trim()))
+    .filter(prop => prop.length > 1)
```

Comment:
```suggestion
    .filter(prop => prop)
```

Reply:
You can just check element, empty string will return `false` when it's converted to boolean